### PR TITLE
feat: release 0.15

### PR DESCRIPTION
**Pull request by abtris**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## 0.15.0
+* fix: change domain from docs.<subdomain>.apiary.io to <subdomain>.docs.apiary.io
```
Comment (opichals) [inline CHANGELOG.md:4]: nit: missing newline

Comment (abtris) [reply CHANGELOG.md]: fixed

**Review by opichals — DISMISSED** [review]
:rocket:

**Comment by abtris** [comment]
@opichals added replacement for http to https

**Comment by abtris** [comment]
@opichals I get security warnings that I made fixes into this PR too.

**Review by opichals — APPROVED** [review]
Thanks!

**Pull request merged**